.github/workflows/lint.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc
         with:
           version: v1.54.2
-          args: --timeout=5m
+          args: --timeout=5m --skip-dirs "internal/third_party/"
```

Comment:
Create a .golangci.yml file instead. Otherwise there will be different behavior when running in CI vs local.

https://github.com/sigstore/gitsign/blob/main/.golangci.yml#L36-L37

Reply:
Done! https://github.com/gittuf/gittuf/pull/137/commits/3a6ea9c47abfb08d3781bb47d0c54eb17ca865ed

I'll add the additional linters separately as that'll need to be accompanied by unrelated-to-this-PR fixes.

Reply:
Replaced that commit with 947ccc82dd60d20a8e9e0f40ce0bfeae8dbdaf17 to fix an issue in the message